<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coinview</class>
 <widget class="QWidget" name="coinview">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>15</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_address">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>20</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_address">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>address：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>900</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_txid">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_txid">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>txid：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>550</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
